DependencyInjection/EkinoDataProtectionExtension.php

```diff
@@ -39,17 +46,50 @@ public function load(array $configs, ContainerBuilder $container)
         if (!$config['encrypt_logs']) {
             $container->removeDefinition('ekino_data_protection.monolog.processor.gdpr');
         }
+
+        if ($config['use_sonata_admin']) {
+            if (!class_exists(SonataAdminBundle::class)) {
+                throw new \RuntimeException('Please add ``sonata-project/admin-bundle`` package and load the bundle in your AppKernel');
```

Comment:
Should be a `LogicException` with message `Please install sonata-project/admin-bundle or turn off use_sonata_admin`